### DISCUSSION TITLE
redist folder not found with v96

### DISCUSSION THREAD

**Discussion by merajhussain** [cefsharp/CefSharp#3930] [General]
Hi @amaitland ,

                    I built cefsharp v96 but I couldn't find  redist folder getting created where I could find the below files

![image](https://user-images.githubusercontent.com/13115973/145796848-55d65aeb-92d6-4377-8e52-21def3fbd90a.png)

From where do i get these files with v96? Could you please help me?

**Comment by amaitland** [comment]
> I built cefsharp v96 but I couldn't find  redist folder getting created where I could find the below files

What do you mean? What did you build exactly? What are you trying to achieve? 

CefSharp is distributed as Nuget packages, all the dependencies are Nuget packages. 

See https://github.com/cefsharp/CefSharp.MinimalExample for a working example.

**Reply by merajhussain** [reply]
thanks @amaitland  with earlier versions a folder used to get created under packages for example cef.redist.x86.93.1.14. I am not able to see such folder with v96
![image](https://user-images.githubusercontent.com/13115973/145928687-1ada26e2-0595-4744-8a05-20bfc8b123a0.png)

**Reply by amaitland** [reply]
The exact same packages are used.

- https://www.nuget.org/packages/cef.redist.x86/
- https://www.nuget.org/packages/cef.redist.x64/

Perhaps you changed to using `PackageReference`? Or upgraded your project file to the new SDK style? Or the `Nuget Restore` failed.

**Reply by merajhussain** [reply]
I think by default its taking sdk style 
![image](https://user-images.githubusercontent.com/13115973/145931175-3121f4be-9612-433d-beb5-8db73b2f432f.png)

should i modify anything in the vcxproject files

**Reply by amaitland** [reply]
Are you building CefSharp from source? If yes for what reason? 

> should i modify anything in the vcxproject files

If you are building CefSharp from source then everything is as expected.

**Reply by amaitland** [reply]
Are you seeing an actual error? There `C#` projects were updated to the newer [SDK Style Project](https://docs.microsoft.com/en-us/nuget/resources/check-project-format) format recently, the project now requires a minimum of `VS2019` to build.
